### PR TITLE
adding "single-file" option to verify command

### PR DESCRIPTION
**Pull request by ptrpfn**
Added 
* implementation to planned subcommand `-sf` of `verify` command
* tests of this subcommand
* adapted README.md accordingly

### CODE REVIEW

**Comment by ptrpfn** [comment]
Don't know why the test in `test_version.py` fails, all tests run successfully locally.

File: ascmhl/commands.py [hunk 1]
```diff
@@ -316,7 +324,9 @@ def verify(
     """
 
     if packing_list is not None:
-        verify_entire_folder(root_path, verbose, packing_list, ignore_list, ignore_spec_file, calculate_only)
+        verify_entire_folder(
```
Comment (jwaggs) [inline ascmhl/commands.py:327]: would be good to make use of the named parameters rather than positional arguments here since there are so many params.

**Review by jwaggs — APPROVED** [review]
@ptrpfn I'm curious if you think it might be worth abstracting out commonalities between verify_directory_hash_subcommand and verify_entire_folder. That being said, I'll approve this PR.

**Comment by ptrpfn** [comment]
Good points @jwaggs . I would propose to merge this PR for now and keep your proposals as issues so that they can be prioritized against other open topics. Will add the issues as soon as possible.

**Pull request merged**